server/app/routes/games.router.js

```diff
@@ -37,9 +36,19 @@ router.get('/:id', function(req, res, next){
 })
 
 router.post('/', function(req, res, next){
+  let invitedPlayers = req.body.players.invited.forEach(u=>u.id);
```

Comment:
Beware, `.forEach` returns `undefined`, not a new array. You're probably looking for `.map` in this case.